.github/mergify.yml

```diff
@@ -13,13 +13,13 @@ pull_request_rules:
     - label!=needs-rebase
     - check-success=DCO
 
-    # If files are changed in .github/, the actionlint check must pass
+    # If workflow configuration files in .github/ are changed, the actionlint check must pass
     - or:
       - and:
         # regex should match the one in .github/workflows/actionlint.yml
-        - files~=.github/.*$
+        - files~=.github/**/*.ya?ml$
         - check-success=actionlint
-      - -files~=.github/.*$
+      - -files~=.github/**/*.ya?ml$
```

Comment:
I don't think `**` is valid here. It's not the same syntax as the github workflows. (Of course ...)

Here we have a Python style regex.  Something like ... `\.github/.*\.ya?ml`

I would have to use the mergify config editor and validate it against some PRs to make sure it's correct, though

Reply:
Ah regex my old friend

Reply:
That config is valid according to the UI dashboard